doc/users.md

```diff
@@ -0,0 +1,29 @@
+[doc](../README.md) > Users
+
+# Users Management
+
+Users information has been introduced in [schema update 07](../server/db/satnogs-07.psql). There's a new table that holds
+user-id, username, salted SHA256 digest, and user role. Until we add flask-admin (or some other management panel), the
+process of adding new users is manual.
+
+To add a new user:
+
+1. log into the server using ssh
+2. psql satnogs
+3. add entry for new user
```

Comment:
One user should be created by default.

Reply:
Default users, especially with passwords set is a security nightmare. See [this question](https://security.stackexchange.com/questions/141157/is-it-dangerous-to-use-default-router-admin-passwords-if-only-trusted-users-are). If you insist on doing that, I can add a user, but it will not have password set by default.

In full blown systems there's usually a mechanism that forces people to change their password on first login on provided passwords. But I don't want to waste time to implement such things here.